tests/test_sourceMatch.py

```diff
@@ -181,6 +181,7 @@ def testPhotometricCalib(self):
             s.setDec(dec * afwGeom.degrees)
             s.set(self.table.getPsfFluxKey(), psfMags[band])
 
+        ifd.close()
```

Comment:
He's in my head too, which is why it was just a suggestion.
I slightly lean towards using the context manager, but it's up to you.